Fix sap gui link sg (#1367)
mulesoft/docs-connectors: sap/5.3/modules/ROOT/pages/sap-connector-config-snc.adoc

To configure an SAP SNC, you must have access to:

> * The SAP GUI application. To install this client, you need JAVA already installed in your machine. Then, download the installer for your OS. An alternative is to open terminal and enter `java -jar PlatinGUI750<OS>.JAR`.

Reviewer: ```suggestion
* The SAP GUI application. To install this client, Java must already be installed on your machine. Then, download the installer for your OS from SAP. 
```

> ** https://sap-infra.s3.amazonaws.com/PlatinGUI750_10-80002494-mac.JAR[Mac 7.50 rev10]

Reviewer: Yes, let's just remove these since we can't access and verify them.
